modules/apps/object/object-storage-salesforce-test/src/testIntegration/java/com/liferay/object/storage/salesforce/internal/rest/manager/v1_0/test/SalesforceObjectEntryManagerImplTest.java

```diff
@@ -170,6 +170,20 @@ public void setUp() throws Exception {
 				_objectDefinition.getObjectDefinitionId()
 			).build());
 
+		ObjectFieldUtil.addCustomObjectField(
+			new DateObjectFieldBuilder(
+			).externalReferenceCode(
+				"format(Due_date__c) FormattedDueDate"
```

Comment:
Hi @jpince,

it seems to not being used as well, could you please remove@

Thanks!